doc/Install_CORTX_on_Supermicro_Servers_with_Seagate_RBOD_Storage.md

```diff
@@ -31,7 +31,7 @@
 
 1. Install [`cortx-os-1.0.0-23.iso`](https://cortxpublic.s3.us-west-2.amazonaws.com/cortx-os-1.0.0-23.iso) **on both servers**. Possible installation options are using a physical USB drive (with installation iso deployed) using PXE environment configured in UEFI mode, using kickstart file `ks.cfg` from the iso. It should be slightly modified to be run in isolated mode (non-production environment). Example configuration:
    ```
```

Comment:
Why do we need to do cat on this file?

Reply:
For the example configuration. Do you think this is unnecessary?